consensus/enclave/impl/src/lib.rs

```diff
@@ -322,6 +322,20 @@ impl SgxConsensusEnclave {
         Ok(transactions)
     }
 
+    /// Validate the provided timestamp is valid.
+    ///
+    /// # Arguments
+    /// * `timestamp` - The timestamp to validate
+    /// * `parent_block_timestamp` - The timestamp of the parent block.
+    fn validate_timestamp(timestamp: u64, parent_block_timestamp: u64) -> Result<u64> {
```

Comment:
might want to require timestamp=0 if the block version does not support timestamps, and only if it does ensure that the timestamp is greater than (or greater-than or equal) to the parent.

Reply:
Conversation brought over to, https://github.com/mobilecoinfoundation/mobilecoin/pull/3733/files#r1399799957